libtenzir/src/config.cpp.in

```diff
@@ -26,15 +26,15 @@ support_for_partition_version(uint64_t partition_version) noexcept {
     = std::array<supported_versions, @TENZIR_PARTITION_VERSION@ +1>{
       // Partition version 0 was introduced implicitly; it's used when no
       // partition version was specified.
-      supported_versions{"Tenzir v1.0", "Tenzir v2.4"},
+      supported_versions{"VAST v1.0", "VAST v2.4"},
       // Alongside partition version 1 we introduced homogeneous partitions and
       // the rebuilder, and required table slices to be of the arrow.v2 encoding
       // in their FlatBuffers table.
-      supported_versions{"Tenzir v2.2", std::nullopt},
+      supported_versions{"VAST v2.2", "Tenzir Node v4.29"},
```

Comment:
While this is technically a breaking change for users that upgrade from VAST v2.2 or VAST v2.3 directly to the next Tenzir Node release, I do not think this is worth mentioning in a changelog entry. It's been too long for that at this point.